fullstack/src/components/container.jsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+import { useState } from "react";
+import Webcam from "@/components/Webcam/Webcam";
+import Widget from "@/components/widget";
+import LineChartComponent from "@/components/graphs/LineChartComponent";
+import BarChartComponent from "@/components/graphs/BarChartComponent";
+
+// Dummy data for the graphs
+const temperatureData = [
```

Comment:
Good job setting up data as props this will likely be the way we want to parse real data